### PR TITLE
feat(badge): add rectangle shaped badge

### PR DESCRIPTION
**Pull request by tomerla**

### CODE REVIEW

File: src/Badge/Badge.js [hunk 1]
```diff
@@ -53,14 +54,18 @@ Badge.propTypes = {
     'T5', 'T5.1'
   ]).isRequired,
 
+  /** set the shape */
+  shape: oneOf(['ellipse', 'rectangle']).isRequired,
```
Comment (lbelinsk) [inline src/Badge/Badge.js]: isRequired is redundant because you set a defaultProp of alipse

File: src/Badge/Badge.spec.js [hunk 2]
```diff
@@ -27,4 +27,8 @@ describe('Badge', () => {
     expect(driver.isOfAppearance('H1')).toBeTruthy();
   });
 
+  it('should allow specifying a shape', () => {
```
Comment (lbelinsk) [inline src/Badge/Badge.spec.js]: I would add a test that checks that the default shape is elipse

File: src/Badge/Badge.js (hunk 1 above)
Comment (tomerla) [inline src/Badge/Badge.js]: agreed
but I just follow the existing pattern -  all other props has a `defaultProp` and they are all marked as `isRequired`
do you want me to change them all?

File: src/Badge/Badge.spec.js (hunk 2 above)
Comment (tomerla) [inline src/Badge/Badge.spec.js]: we have already a e2e tests for it
but sure.. why not :)

**Comment by tomerla** [comment]
@lbelinsk so I updated a test for a default ellipse shape and updated the docs
please let me know what do you want to do with the `isRequired` (wrote to you in the line comment)
tnx!

File: src/Badge/Badge.js (hunk 1 above)
Comment (lbelinsk) [inline src/Badge/Badge.js]: Yeah, it will be great! I guess it is a trailing error :)

**Comment by tomerla** [comment]
tnx @lbelinsk !

**Pull request merged**